packages/firestore/test/unit/local/remote_document_cache.test.ts

```diff
@@ -216,3 +240,21 @@ function genericRemoteDocumentCacheTests(
     assertMatches([], changedDocs);
   });
 }
+
+function assertMatches(
+  expected: MaybeDocument[],
+  actual: MaybeDocumentMap
+): void {
+  expect(actual.size).to.equal(expected.length);
+  actual.forEach((actualKey, actualDoc) => {
+    const found = expected.find(expectedDoc => {
+      if (actualKey.isEqual(expectedDoc.key)) {
+        expectEqual(actualDoc, expectedDoc);
+        return true;
+      }
+      return false;
+    });
+
+    expect(found).to.not.be.undefined;
```

Comment:
is there no `.to.be.defined` ?

Reply:
But.... this is just code that I moved around.

Changed it to `expect(found).to.exist`